packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/components/NameAndWebUiLinkDetailsItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { EditProviderUIModal, useModal } from 'src/modules/Providers/modals';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ExternalLink } from '@kubev2v/common';
+import { V1beta1Provider } from '@kubev2v/types';
+import { DescriptionListDescription } from '@patternfly/react-core';
+
+import { DetailsItem } from '../../../../../utils';
+
+import { ProviderDetailsItemProps } from './ProviderDetailsItem';
+
+/**
+ * @typedef {Object} NameDetailsItemProps - extends ProviderDetailsItemProps
+
+ * @property {string} [webUILinkText - A text to be displayed as a content.
+ * @property {function} [calcWebUILink] - A method for auto calculating the provider's UI path.
+ **/
+
+export interface NameAndWebUiLinkDetailsItemProps extends ProviderDetailsItemProps {
+  webUILinkText?: string;
+  calcWebUILink?: (provider: V1beta1Provider) => string;
+}
+
+const getProviderUIAnnotation = (provider: V1beta1Provider): string =>
+  provider?.metadata?.annotations?.['forklift.konveyor.io/providerUI'];
+
+/**
+ * Component for displaying the provider name details item.
+ * It includes a title and two content items:
+ * 1. The provider name (non-editable)
+ * 2. The provider web UI link (editable, can be auto calculated or taken from YAML)
+ *
+ * @component
+ * @param {DetailsItemProps} props - The props of the details item.
+ */
+export const NameAndWebUiLinkDetailsItem: React.FC<NameAndWebUiLinkDetailsItemProps> = ({
```

Comment:
```suggestion
export const NameAndLinkDetailsItem: React.FC<NameAndWebUiLinkDetailsItemProps> = ({
```

Reply:
The name is long but just `Link` is not self explained...if you insist on a shorter name then I prefer `NameUiLinkDetailsItem`

Reply:
ok, I think the "and" is important, so `NameAndUiLinkDetailsItem` WDYT ?